index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>PORTFOLIO</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
+  </head>
+  <body>
+    <div class="hero">
+      <nav>
+        <h2 class="logo"><span> Welcome</span></h2>
+        
+        <ul class = "nav-list">
+          <li><a href="#">Home</a></li>
+          <li><a href="#">About me</a></li>
+          <li><a href="#">Skills</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+        <a href="#" class="btn"> Let's connect</a>
+        <label for="check" class="hambuger-btn">
+          <i class = "fas fa-bars"></i>
+        </label>
+      </nav>
+      <div class="content">
+        <h1>Hey there. I'm Esther.</h1>
+        <h2>I'm a software developer</h2>
```

Comment:
- [ ] Kindly make sure you use the correct styling on the above text like on the Figma design e.g.: `color`, `fonts(size && weight)`, `line height`, `paddings && margins`,...

| Figma Design | Your DEsign |
| ---------------- | -------------------- |
| ![Screenshot from 2022-09-27 20-22-30](https://user-images.githubusercontent.com/83042742/192608154-8ac2b24a-708f-48d2-be81-d34d239287d5.png) | ![Screenshot from 2022-09-27 20-33-00](https://user-images.githubusercontent.com/83042742/192608553-da857f9c-b0ed-440a-8c01-be826a1805c1.png) |